x-pack/plugins/stack_connectors/server/routes/get_inference_services.ts

```diff
@@ -32,15 +33,676 @@ export const getInferenceServicesRoute = (router: IRouter) => {
     req: KibanaRequest<unknown, unknown, unknown>,
     res: KibanaResponseFactory
   ): Promise<IKibanaResponse> {
-    const esClient = (await ctx.core).elasticsearch.client.asInternalUser;
+    // Temporarily hard-coding the response until the real implementation is ready with the updated response - https://github.com/elastic/ml-team/issues/1428
 
-    const response = await esClient.transport.request<{
-      endpoints: InferenceProvider[];
-    }>({
-      method: 'GET',
-      path: `/_inference/_services`,
-    });
+    // const esClient = (await ctx.core).elasticsearch.client.asInternalUser;
+
+    // // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    // const response = await esClient.transport.request<any[]>({
+    //   method: 'GET',
+    //   path: `/_inference/_services`,
+    // });
+
+    const response: InferenceProvider[] = [
+      {
+        service: 'cohere',
+        name: 'Cohere',
+        description: '',
+        task_types: ['text_embedding', 'rerank', 'completion'],
+        configurations: {
+          api_key: {
+            default_value: null,
+            description: `API Key for the provider you're connecting to.`,
+            label: 'API Key',
+            required: true,
+            sensitive: true,
+            updatable: true,
```

Comment:
`Updatable` is in the RFC so I added it to the hard-coded response schema but just set it to `true` for now since it's not used anywhere in the UI. When I update the API on the Elasticsearch side, I will verify the updatability of each field and set appropriately.